tasks/configure-Debian.yml

```diff
@@ -38,6 +38,17 @@
   notify: restart apache
   when: apache_create_vhosts
 
+- name: Configure Apache (Add listen port for every vhost).
+  blockinfile:
+    dest: "{{ apache_server_root }}/conf/{{ apache_daemon }}.conf"
+    insertafter: "^Listen {{ apache_listen_port }}"
+    block: "{% set return = joiner(\"\\n\") %}{% for item in apache_vhosts %}{% if item.listenport is defined and apache_listen_port != item.listenport %}{{ return() }}Listen {{ item.listenport }}{% endif %}{% endfor %}"
```

Comment:
I have introduce an the error on vhost.conf.j2 if you don't use the "vhost.listenport". I correct it, sorry for this.